app/routes/items/history.js

```diff
@@ -1,15 +1,34 @@
+import Ember from "ember";
 import AuthorizeRoute from "./../authorize";
 
 export default AuthorizeRoute.extend({
   model(params) {
-    return this.store.findRecord("item", params.item_id, { reload: true });
+    const queryParams = {
+      package_id: params.item_id,
+      per_page: 100
+    };
+    return Ember.RSVP.hash({
+      item: this.store.findRecord("item", params.item_id, { reload: true }),
+      itemActions: this.store.query("item_action", queryParams)
```

Comment:
@abulsayyad123 Could you investigate how much we're using the term `item_action` in the project ? I really think `inventory_action` makes a lot more sense. I'm hoping to refactor it